WeakAuras/Prototypes.lua

```diff
@@ -2431,7 +2431,7 @@ Private.event_prototypes = {
         store = true,
         conditionType = "select",
         enable = function(trigger)
-          return WeakAuras.IsCataOrRetail() and (trigger.unit == "group" or trigger.unit == "raid" or trigger.unit == "party")
+          return trigger.unit == "group" or trigger.unit == "raid" or trigger.unit == "party"
```

Comment:
I think we can lift this unit restriction to make it work with player & target

Reply:
I agree, I'll put that into the next PR.